code/src/java/pcgen/gui2/facade/TodoFacadeImpl.java

```diff
@@ -33,6 +29,7 @@
  * 
  * @author James Dempsey &lt;jdempsey@users.sourceforge.net&gt;
  */
+@SuppressWarnings("TodoComment")
```

Comment:
IntelliJ in particular gets annoyed by the text string "todo". This is useless when you're writing a TODO class.